chainer/datasets/cifar.py

```diff
@@ -0,0 +1,144 @@
+import os
+import tarfile
+
+import numpy
+import six.moves.cPickle as pickle
+
+from chainer.dataset import download
+from chainer.datasets import tuple_dataset
+
+
+def get_cifar10(withlabel=True, ndim=3, scale=1.):
+    """Gets the CIFAR-10 dataset.
+
+    `CIFAR-10 <https://www.cs.toronto.edu/~kriz/cifar.html>`_ is a set of small
+    natural images. Each example is an RGB color image of size 32x32,
+    classified into 10 groups. Each pixel is scaled to values in the interval
```

Comment:
same as MNIST dataset (This function scales the pixels to floating point values in the interval ``[0, scale]``)